README.md

```diff
@@ -8,29 +8,29 @@
 Go source for [deoplete.nvim](https://github.com/Shougo/deoplete.nvim) use [gocode](https://github.com/nsf/gocode).
 
 ## Overview
-Asynchronous Go completion for Neovim.  
```

Comment:
Ah, sorry but not nits, last two spaces need to for the new line.

Reply:
It means, Do not trim the two spaces.

Reply:
Why?  I don't understand the two speces reason.

Reply:
Oh, it's markdown(GitHub Flavor Markdown) specific. See below diff.
- https://github.com/zchee/deoplete-go/blob/master/README.md
- https://github.com/zchee/deoplete-go/blob/c1f372aa08f423ecb09d0a46726e96dde059674c/README.md

There is new line before the `Use,` word.

Reply:
Ah, I get it.

Reply:
🙇